go/sqlescape/ids.go

```diff
@@ -44,3 +44,12 @@ func EscapeIDs(identifiers []string) []string {
 	}
 	return result
 }
+
+// UnescapeID reverses any backticking in the input string.
+func UnescapeID(in string) string {
```

Comment:
Do we care about leading or trailing whitespace for idents? I'm not sure that's possible, but came to mind.

Reply:
I would think the parser would provide identifiers without whitespaces, but don't know for sure.

Reply:
Yeah, I would think so too... we can leave it as-is.